packages/amplication-client/src/VersionControl/BuildPage.scss

```diff
@@ -4,6 +4,25 @@ $log-min-width: 360px;
 $icon-size: 20px;
 
 .build-page {
+  &__header {
+    gap: 24px;
+    margin-bottom: 16px;
```

Comment:
Please don't use hard-coded values.
For the margin `16px` you can use `var(--large-spacing)` if I'm not mistaken.
For the gap you can set a local var like in line 4.

Reply:
Resolved with `var(--large-spacing)` for `16px` and `var(--double-spacing)` for `24px`.